ixwebsocket/IXSocketOpenSSL.cpp

```diff
@@ -114,14 +114,17 @@ namespace ix
         SSL_CTX* ctx = SSL_CTX_new(_ssl_method);
         if (ctx)
         {
```

Comment:
shouldn't we have an else block here, that use SSL_VERIFY_NONE ? or is that not needed ?

Reply:
I think that's default (0), but it's better to be clear; fixed